packages/@react-stately/collections/docs/collections.mdx

```diff
@@ -0,0 +1,387 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-stately/collections';
+import dataDocs from 'docs:@react-stately/data';
+import {HeaderInfo, PropTable, TypeLink} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+---
+category: Collections
+---
+
+# Collection Components
+
+Many components display a collection of items, and provide functionality such as keyboard 
+navigation, selection, and more. React Spectrum has a unified API to define the items
```

Comment:
Do we want to refer to it as "React Spectrum" here? This page lives within the React Stately docs, but I assume will be referenced by users of RSP as well...